{{cookiecutter.project_name}}/README.md

```diff
@@ -0,0 +1,47 @@
+<p align="center">
+  <img src="https://raw.githubusercontent.com/dbt-labs/dbt/ec7dee39f793aa4f7dd3dae37282cc87664813e4/etc/dbt-logo-full.svg" alt="dbt logo" width="500"/>
+</p>
+
+**[dbt](https://www.getdbt.com/)** enables data analysts and engineers to transform their data using the same practices that software engineers use to build applications.
+
+dbt is the T in ELT. Organize, cleanse, denormalize, filter, rename, and pre-aggregate the raw data in your warehouse so that it's ready for analysis.
+
+## {{ cookiecutter.project_name }}
+This repo contains the base code to help you start to build out your {{ cookiecutter.project_name }} adapter plugin, for more information on how to build out the adapter please follow the [docs](https://docs.getdbt.com/docs/contributing/building-a-new-adapter)
+
+ ** Note ** this `README` is meant to be replaced with what information would be required to use your adpater once your at a point todo so.
```

Comment:
Per https://github.com/dbt-labs/dbt-database-adapter-scaffold/pull/24#issuecomment-1100389318: Let's add a note here, by default, that says something like:

> ### Versioning

> This adapter plugin follows [semantic versioning](https://semver.org/). The first version of this plugin is v{{ cookiecutter.project_version }}, in order to be compatible with dbt Core v{{ cookiecutter.dbt_version }}.
> 
> It's also brand new! For {{ cookiecutter.adapter_name }}-specific functionality, we will aim for backwards-compatibility wherever possible. We are likely to be iterating more quickly than most major-version-1 software projects. To that end, backwards-incompatible changes will be clearly communicated and limited to minor versions (once every three months).

Reply:
cc @ChenyuLInx